tests/compiler/cbs/accumulator_for.cpp

```diff
@@ -10,31 +10,31 @@
 #include <array>
 #include <iostream>
 
-#include <CL/sycl.hpp>
+#include <SYCL/sycl.hpp>
```

Comment:
Please use lower-case `<sycl/sycl.hpp>`.

Early versions of the SYCL 2020 specification used `<SYCL/sycl.hpp>`, which is why AdaptiveCpp supports both - but current SYCL 2020 only has `<sycl/sycl.hpp>`.

Please change this everywhere :)